sdk/main/src/impl/NodeAddress.cc

```diff
@@ -75,11 +193,11 @@ std::string NodeAddress::toString() const
     {
       if (counter == 0)
       {
-        outputStream << endpoint.toString();
+        outputStream << endpoint->toString();
       }
       else
       {
-        outputStream << std::setw(columnWidth) << "" << endpoint.toString();
+        outputStream << std::setw(columnWidth) << "" << endpoint->toString();
```

Comment:
Just curious, does add the empty string here actually do anything?

Reply:
I think this empty string has been left like that in some previous commit - you can check the version of [NodeAddress](https://github.com/hashgraph/hedera-sdk-cpp/blob/d28771d8f78d328773cc5037865c40d07c5efdc0/sdk/main/src/impl/NodeAddress.cc#L82) from the `main` branch.
Anyways I just removed it in 36be04f972795aabed09f10c1bc6bf2441694fa3.